redis-7.0/0000-Disable-protected-mode.patch

```diff
@@ -0,0 +1,29 @@
+From b91e39701c7a71f229be34dcf078579d37367436 Mon Sep 17 00:00:00 2001
```

Comment:
If these are all the same, then we can create it once and use directory symlinks for each of the copies.